### DISCUSSION TITLE
Which file manage are we using

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [end-4/dots-hyprland#545] [General]
Hey i am new to linux and Hyperland so i just wanted to know which file manager we are using. i installed dolphin but i think i have to theme it separately. The one that pops up when i am choosing my wallpaper or downloading things, which one is that?

**Comment by end-4** [comment]
that's not a file manager but the xdg desktop portal file picker

**Reply by end-4** [reply]
if you want similar ones, try gtk file managers such as nautilus, nemo, and thunar

**Reply by AKA-Akhil** [reply]
thanks

**Reply by binarykitchen** [reply]
Hmm, which one is the best looking? Thunar looks ugly.

**Reply by end-4** [reply]
nautilus is the nicest looking one
it's also the slowest and buggiest though
but i still use it

**Reply by binarykitchen** [reply]
I still cannot decide - maybe we overthink :)
